templates/profile.html

```diff
@@ -14,11 +14,30 @@ <h4>Funds</h4>
     <div>
         <ol>
         {% for fund in funds %}
-            <li><a href="{{ url_for('fund_api.fund', fund_id=fund.id) }}">{{ fund.fund_name }}</a></li>
+            <li><a href="{{ url_for('fund_api.fund', fund_id=fund.id) }}">{{ fund.fund_manager }} > {{ fund.fund_name }} > {{ fund.fund_vintage }} > {{ fund.fund_nav }} > {{ fund.fund_unfunded }}</a></li>
```

Comment:
This looks temporary and should be removed

Reply:
This is a preparation for a new page look.

Reply:
Agreed